docs/marionette.application.md

```diff
@@ -160,6 +160,21 @@ allows you to provide extra configuration for various parts of your app througho
 initialization sequence.
 
 ```javascript
+var App = Mn.Application.extend({
+  initialize: function(options) {
+    console.log('Initialize');
+  },
+
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(options.data); 
```

Comment:
Hi @denar90 , the second example should use options.data instead to show how to receive options passed in from app.start. Notice this example passes options in app.start instead of var app = new App(options); as in first example.

app.start({
  data: {
    id: 1,
    text: 'value'
  }
});